src/convert.js

```diff
@@ -11,13 +19,13 @@ function highlightText(sentenceText) {
       if (hasNumber.test(word)) {
         return word;
       }
-      const { length } = word;
-      let midPoint = 1;
-      if (length > 3) midPoint = Math.round(length / 2);
+
+      const midPoint = determineMidpoint(word);
```

Comment:
can skip with `word.length <= 3`, we only want to bold the first letter of these words

Reply:
I'm not convinced this should be hardcoded, if change fixation is implemented then we should let the algorithm decide, no?
The difference is pretty much `Math.ceil(x)` vs. `Math.floor()` which will produce the difference of 1 character that we see here on 3 letter words. Strong fixation would be 2 letters bolded and weak is 1 letters.

Reply:
> Strong fixation would be 2 letters bolded and weak is 1 letters.

Maybe that is something that could be delegated to a setting of the chrome extension?